mlflow/server/js/src/experiment-tracking/components/ExperimentViewUtil.test.js

```diff
@@ -177,8 +177,9 @@ describe('ExperimentViewUtil', () => {
         idx: 1,
         isParent: true,
         runId: 2,
+        level: 0
       },
-      { hasExpander: false, idx: 0, isParent: false },
+      { hasExpander: false, idx: 0, isParent: false, level: 1},
```

Comment:
Can we add a test for verifying multiple levels and not just level 1.

Reply:
Added to level 2.

Do you think we need more?